website/docs/sdk-reference/elixir.md

```diff
@@ -0,0 +1,251 @@
+---
+id: elixir
+title: Elixir
+---
+## Getting started:
+### 1. Add `configcat` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:configcat, "~> 1.0.0"}
+  ]
+end
+```
+
+### 2. Add `ConfigCat` to your application Supervisor tree:
+
+```elixir
+def start(_type, _args) do
+  children = [
+    {ConfigCat, [sdk_key: "YOUR SDK KEY"]},
+    MyApp
+  ]
+
+  opts = [strategy: :one_for_one, name: Simple.Supervisor]
+  Supervisor.start_link(children, opts)
+end
+```
+
+### 4. Get your setting value
+```elixir
+isMyAwesomeFeatureEnabled = ConfigCat.get_value("isMyAwesomeFeatureEnabled", false)
+if isMyAwesomeFeatureEnabled do
+    do_the_new_thing()
+else
+    do_the_old_thing()
+end
+```
+
+## Configuring the *ConfigCat Client*
+*ConfigCat Client* is responsible for:
+- managing the communication between your application and ConfigCat servers.
+- caching your setting values and feature flags.
+- serving values quickly in a failsafe way.
+
+`{ConfigCat, options}` returns a client with default options.
+
+| Properties | Description                                                                                               |
+| ---------- | --------------------------------------------------------------------------------------------------------- |
+| `sdk_key`  | **REQUIRED.** SDK Key to access your feature flags and configurations. Get it from *ConfigCat Dashboard*. |
+| `data_governance`  | Optional, defaults to `ConfigCat.DataGovernance.global()`. Describes the location of your feature flag and setting data within the ConfigCat CDN. This parameter needs to be in sync with your Data Governance preferences. [More about Data Governance](advanced/data-governance.md). Available options: `DataGovernance.global()`, `DataGovernance.eu_only()`. |
```

Comment:
If you approve https://github.com/configcat/elixir-sdk/pull/55, this will have to change to use `:global` or `:eu_only` instead of `DataGovernance.global()` and `DataGovernance.eu_only()`.

Reply:
Done 👍